demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1267,6 +1267,10 @@ export class DemoMeetingApp
     }
   }
 
+  audioInputMuteStateChanged(device: string | MediaStream, muted: boolean): void {
+    this.log('Mute state: device', device, muted ? 'is muted' : 'is not muted');
```

Comment:
Should we also update the mic button to show muted?

Reply:
A good follow-up is to make the mic button show muted _and be un-mutable_, and show an explanatory tooltip. I didn't have time to address those: I did the bare minimum demo work to allow us to tell whether the feature works.